src/functions-reference/covariance_matrix_distributions.Rmd

```diff
@@ -110,6 +111,52 @@ lower-triangular Cholesky factor of the scale matrix `L_S`; may only be used in
 transformed data and generated quantities blocks
 `r since("2.30")`
 
+## Inverse Wishart distribution, Cholesky Parameterization (#inverse-wishart-cholesky-distribution)
+
+### Probability density function
+
+If $K \in \mathbb{N}$, $\nu \in (K-1, \infty)$, and $L_S \in
+\mathbb{R}^{K \times K}$ is lower triangular and such that $L_S L_S^{\top}$
+is positive definite,then for
+$L_W \in \mathbb{R}^{K \times K}$ is lower triangular and such that $L_WL_W^{\top}$
+is positive definite, \[
+\text{InvWishartCholesky}(L_W \mid \nu,L_S) =
```

Comment:
I'd like to see inv Wishart before inv Wishart Cholesky as a section.  Then I'd like to see the top index in the same order.  It's currently in this order:

* Wishart
* Wishart Cholesky
* inv Wishart Cholesky
* inv Wishart